app/scripts/controllers/metametrics.js

```diff
@@ -540,6 +540,9 @@ export default class MetaMetricsController {
         (rpc) => rpc.chainId,
       ),
       [TRAITS.THREE_BOX_ENABLED]: metamaskState.threeBoxSyncingAllowed,
+      [TRAITS.NUMBER_OF_NFT_COLLECTIONS]: this._getNumberOfNFtCollection(
+        metamaskState,
+      ),
```

Comment:
Should we only track `number_of_nft_collections` if `metamaskState.useCollectibleDetection` is enabled?

Reply:
I don't think so....collectibles can be added manually too.